flake.nix

```diff
@@ -17,26 +16,26 @@
   };
 
   outputs = inputs@{ flake-parts, nixpkgs, crane, fenix, ... }:
-    flake-parts.lib.mkFlake { inherit inputs; } {
-      systems = [
-        # systems for which you want to build the `perSystem` attributes
-        "x86_64-linux"
-        "x86_64-darwin"
-        "aarch64-linux"
-        "aarch64-darwin"
-      ];
+    let
+      # Set version numbers as variables
+      edgedbServerVersion = {
+        major = 5;
+        minor = 3;
+      };
+      edgedbCliVersion = "v5.1.0";
```

Comment:
Just changing the cli version here would not work, because the sha256 needs to be updated.

I think having the version down next to sha is good enough.